sql/catalyst/src/test/scala/org/apache/spark/sql/RowTest.scala

```diff
@@ -121,10 +121,6 @@ class RowTest extends FunSpec with Matchers {
       externalRow should be theSameInstanceAs externalRow.copy()
     }
 
-    it("copy should return same ref for internal rows") {
```

Comment:
Copy returned the same ref because it is supposed to be immutable. See https://github.com/apache/spark/pull/10553 for more context.